buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -0,0 +1,29 @@
+object VersionsBackend {
```

Comment:
Why `backend`?

Reply:
totally forgot to remove the actual sources.  in the buildSrc. buildSrc is needed becasue dokka only works from plugins (such as our conventions, or buildSrc) for multi-module projects)